ext/nnstreamer/tensor_decoder/tensordec-imagesegment.c

```diff
@@ -238,15 +238,59 @@ is_getTransformSize (void **pdata, const GstTensorsConfig * config,
 static void
 set_color_according_to_label (image_segments * idata, GstMapInfo * out_info)
 {
-
+  uint32_t *frame = (uint32_t *) out_info->data;
+  uint32_t *pos;
+  int i, j;
+  const uint32_t label_color[21] = {
+    0xFF000040, 0xFF800000, 0xFFFFEFD5, 0xFF40E0D0, 0xFFFFA500,
+    0xFF00FF00, 0xFFDC143C, 0xFFF0F8FF, 0xFF008000, 0xFFEE82EE,
+    0xFF808080, 0xFF4169E1, 0xFF008080, 0xFFFF6347, 0xFF000000,
+    0xFFFF4500, 0xFFDA70D6, 0xFFEEE8AA, 0xFF98FB98, 0xFFAFEEEE,
+    0xFFFFF5EE
+  };
+
+  for (i = 0; i < idata->height; i++) {
+    for (j = 0; j < idata->width; j++) {
+      int label_idx = idata->segment_map[i][j];
+      pos = &frame[i * idata->width + j];
+      if(label_idx == 0) continue; // Do not set color for background
```

Comment:
Please reformat with gst-indent script.

Reply:
Also please fix line comment (c-std c89, use ```/* */```, do not use ```//```)